services/motion/builtin/move_request.go

```diff
@@ -268,24 +222,33 @@ func (mr *moveRequest) obstaclesIntersectPlan(
 				return state.ExecuteResponse{}, err
 			}
 
+			// build representation of frame system's inputs
 			currentInputs, err := mr.kinematicBase.CurrentInputs(ctx)
 			if err != nil {
 				return state.ExecuteResponse{}, err
 			}
+			inputMap := referenceframe.StartPositions(mr.planRequest.FrameSystem)
+			inputMap[mr.kinematicBase.Name().ShortName()] = currentInputs
 
 			// get the pose difference between where the robot is versus where it ought to be.
-			errorState, err := mr.kinematicBase.ErrorState(ctx, waypoints, waypointIndex)
+			errorState, err := mr.kinematicBase.ErrorState(ctx, plan, waypointIndex)
 			if err != nil {
 				return state.ExecuteResponse{}, err
 			}
 
+			// check no obstacles intersect the portion of the plan which has yet to be executed
+			remainingPlan, err := motionplan.RemainingPlan(plan, waypointIndex)
+			if err != nil {
+				return state.ExecuteResponse{}, err
+			}
+			mr.logger.Debug(plan.Path())
```

Comment:
Do we want to keep this debug statement here?

Reply:
i didnt mean to have this thanks